test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
- Xóa toàn bộ file test trong thư mục test
- Thêm dòng /test/* vào gitignore để bỏ file test trước khi push lên repo